src/main/java/apoc/periodic/Periodic.java

```diff
@@ -163,6 +163,13 @@ private long executeNumericResultStatement(@Name("statement") String statement,
         return Stream.of(info);
     }
 
+    @Procedure(mode = Mode.WRITE)
+    @Description("apoc.periodic.repeat2('name',statement,repeat-rate-in-seconds, config) submit a repeatedly-called background statement with configuration")
+    public Stream<JobInfo> repeat2(@Name("name") String name, @Name("statement") String statement, @Name("rate") long rate, @Name("config") Map<String,Object> config ) {
+        JobInfo info = schedule(name, () -> Iterators.count(db.execute(statement, config)),0,rate);
```

Comment:
don't call execute with the config directly, either rename it to `params`
or add a `params` entry into `config`

I would want the 2nd.

Reply:
@jexp Okay so preference is pass in config.params?

Reply:
Ok, so similar to how apoc.periodic.iterate does it.